jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/SNMP.java

```diff
@@ -17,14 +17,17 @@
 
 import java.text.ParseException;
 import java.time.ZonedDateTime;
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
 
 import us.dot.its.jpo.ode.model.OdeObject;
-import us.dot.its.jpo.ode.util.CodecUtils;
 import us.dot.its.jpo.ode.util.DateTimeUtils;
 
 public class SNMP extends OdeObject {
    private static final long serialVersionUID = 6622977652181526235L;
 
+   private static final Logger logger = LoggerFactory.getLogger(SNMP.class);

```

Comment:
We can probably remove this line since the logger isn't invoked. No real big deal either way.

Reply:
Good call, we had added it in for testing but no longer need it. I'll update the PR and get it back over to you.